source/_layouts/post.blade.php

```diff
@@ -59,7 +59,7 @@
             <div class="col-lg-4">
               <div class="ud-blog-sidebar">
                 <div class="ud-articles-box">
-                  <h3 class="ud-articles-box-title">Last Articles</h3>
+                  <h3 class="ud-articles-box-title">{{ $page->t("Last Articles") }}</h3>
```

Comment:
Is good to only use double quote in PHP if you will use interpolation (put a var inside the string).

```suggestion
                  <h3 class="ud-articles-box-title">{{ $page->t('Last Articles') }}</h3>
```